lib/apiservers/engine/backends/container.go

```diff
@@ -1431,3 +1473,72 @@ func copyEscapable(dst io.Writer, src io.ReadCloser, keys []byte) (written int64
 func clientFriendlyContainerName(name string) string {
 	return fmt.Sprintf("/%s", name)
 }
+
+func processVolumeParam(volString string) (volumeFields, error) {
+	volumeStrings := strings.Split(volString, ":")
+	fields := volumeFields{}
+	switch len(volumeStrings) {
```

Comment:
I think this deserves a comment at least.  Just include what the expected invocations are and the resulting params.